<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>563</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>561</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cadastro&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #444444</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>204</width>
        <height>44</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: &quot;white&quot;
}</string>
      </property>
      <property name="text">
       <string>ID Departamento:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_nome">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: &quot;white&quot;;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>73</width>
        <height>44</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: &quot;white&quot;
}</string>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_idade">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: &quot;white&quot;;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_cpf">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>180</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: &quot;white&quot;;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_gravar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #100E19;
}
QPushButton::default{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #FFFFFF;
	padding: 2px;
	background-color: #151a1e;
}
QPushButton:hover{
	border-style: solid;
	border-top-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #C0DB50, stop:0.4 #C0DB50, stop:0.5 #100E19, stop:1 #100E19);
    border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #100E19, stop:0.5 #100E19, stop:0.6 #C0DB50, stop:1 #C0DB50);
    border-left-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #C0DB50, stop:0.3 #C0DB50, stop:0.7 #100E19, stop:1 #100E19);
    border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #C0DB50, stop:0.3 #C0DB50, stop:0.7 #100E19, stop:1 #100E19);
	border-width: 2px;
    border-radius: 1px;
	color: #d3dae3;
	padding: 2px;
}
QPushButton:pressed{
	border-style: solid;
	border-top-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #d33af1, stop:0.4 #d33af1, stop:0.5 #100E19, stop:1 #100E19);
    border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #100E19, stop:0.5 #100E19, stop:0.6 #d33af1, stop:1 #d33af1);
    border-left-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #d33af1, stop:0.3 #d33af1, stop:0.7 #100E19, stop:1 #100E19);
    border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #d33af1, stop:0.3 #d33af1, stop:0.7 #100E19, stop:1 #100E19);
	border-width: 2px;
    border-radius: 1px;
	color: #d3dae3;
	padding: 2px;
}</string>
      </property>
      <property name="text">
       <string>Gravar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>52</width>
        <height>44</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: &quot;white&quot;
}</string>
      </property>
      <property name="text">
       <string>CPF:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_id_departamento">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>270</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: &quot;white&quot;;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>69</width>
        <height>44</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: &quot;white&quot;
}</string>
      </property>
      <property name="text">
       <string>Idade:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Consulta</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>521</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget{
	color:#DCDCDC;
	background:#444444;
	border:1px solid #242424;
	alternate-background-color:#525252;
	gridline-color:#242424;
}

QTableWidget::item:selected{
	color:#DCDCDC;
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}


QTableWidget::item:hover{
	background:#5B5B5B;
}

QHeaderView::section{
	text-align:center;
	background:#5E5E5E;
	padding:3px;
	margin:0px;
	color:#DCDCDC;
	border:1px solid #242424;
	border-left-width:0;
}

QScrollBar:vertical{
	background:#484848;
	padding:0px;
	border-radius:6px;
	max-width:12px;
}


QScrollBar::handle:vertical{
	background:#CCCCCC;
}

QScrollBar::handle:hover:vertical,QScrollBar::handle:pressed:vertical{
	background:#A7A7A7;
}

QScrollBar::sub-page:vertical{
	background:444444;
}

QScrollBar::add-page:vertical{
	background:5B5B5B;
}


QScrollBar::add-line:vertical{
	background:none;
}

QScrollBar::sub-line:vertical{
	background:none;
}</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>177</number>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>NOME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CPF</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IDADE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DEPARTAMENTO</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="bt_atualizar">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>170</y>
        <width>111</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #100E19;
}
QPushButton::default{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #FFFFFF;
	padding: 2px;
	background-color: #151a1e;
}
QPushButton:hover{
	border-style: solid;
	border-top-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #C0DB50, stop:0.4 #C0DB50, stop:0.5 #100E19, stop:1 #100E19);
    border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #100E19, stop:0.5 #100E19, stop:0.6 #C0DB50, stop:1 #C0DB50);
    border-left-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #C0DB50, stop:0.3 #C0DB50, stop:0.7 #100E19, stop:1 #100E19);
    border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #C0DB50, stop:0.3 #C0DB50, stop:0.7 #100E19, stop:1 #100E19);
	border-width: 2px;
    border-radius: 1px;
	color: #d3dae3;
	padding: 2px;
}
QPushButton:pressed{
	border-style: solid;
	border-top-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #d33af1, stop:0.4 #d33af1, stop:0.5 #100E19, stop:1 #100E19);
    border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #100E19, stop:0.5 #100E19, stop:0.6 #d33af1, stop:1 #d33af1);
    border-left-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #d33af1, stop:0.3 #d33af1, stop:0.7 #100E19, stop:1 #100E19);
    border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #d33af1, stop:0.3 #d33af1, stop:0.7 #100E19, stop:1 #100E19);
	border-width: 2px;
    border-radius: 1px;
	color: #d3dae3;
	padding: 2px;
}</string>
      </property>
      <property name="text">
       <string>Atualizar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
